spreadsheets/collision_rank.yaml

```diff
@@ -8,7 +8,25 @@
 - {$layer: 'water', kind: 'ocean'}
 - {$layer: 'places', kind: 'country'}
```

Comment:
We also do some population grading of `country` features client side, so let's also include that in this PR, please.

Something like:

```
- {$layer: 'places', kind: 'country', population_rank: 18}
- {$layer: 'places', kind: 'country', population_rank: 17}
- {$layer: 'places', kind: 'country', population_rank: 16}
- {$layer: 'places', kind: 'country', population_rank: 15}
- {$layer: 'places', kind: 'country', population_rank: 14}
- {$layer: 'places', kind: 'country', population_rank: 13}
- {$layer: 'places', kind: 'country', population_rank: 12}
- {$layer: 'places', kind: 'country', population_rank: 11}
- {$layer: 'places', kind: 'country', population_rank: 10}
- {$layer: 'places', kind: 'country', population_rank: 9}
- {$layer: 'places', kind: 'country', population_rank: 8}
- {$layer: 'places', kind: 'country', population_rank: 7}
- {$layer: 'places', kind: 'country', population_rank: 6}
- {$layer: 'places', kind: 'country', population_rank: 5}
- {$layer: 'places', kind: 'country', population_rank: 4}
- {$layer: 'places', kind: 'country', population_rank: 3}
- {$layer: 'places', kind: 'country', population_rank: 2}
- {$layer: 'places', kind: 'country', population_rank: 1}
- {$layer: 'places', kind: 'country'}
```

Reply:
Please also do for `region` (lower down):

```
- {$layer: 'places', kind: 'region', population_rank: 18}
- {$layer: 'places', kind: 'region', population_rank: 17}
- {$layer: 'places', kind: 'region', population_rank: 16}
- {$layer: 'places', kind: 'region', population_rank: 15}
- {$layer: 'places', kind: 'region', population_rank: 14}
- {$layer: 'places', kind: 'region', population_rank: 13}
- {$layer: 'places', kind: 'region', population_rank: 12}
- {$layer: 'places', kind: 'region', population_rank: 11}
- {$layer: 'places', kind: 'region', population_rank: 10}
- {$layer: 'places', kind: 'region', population_rank: 9}
- {$layer: 'places', kind: 'region', population_rank: 8}
- {$layer: 'places', kind: 'region', population_rank: 7}
- {$layer: 'places', kind: 'region', population_rank: 6}
- {$layer: 'places', kind: 'region', population_rank: 5}
- {$layer: 'places', kind: 'region', population_rank: 4}
- {$layer: 'places', kind: 'region', population_rank: 3}
- {$layer: 'places', kind: 'region', population_rank: 2}
- {$layer: 'places', kind: 'region', population_rank: 1}
- {$layer: 'places', kind: 'region'}
```

Reply:
Fixed in 646eec7.